cmd/apiserver/app/server/run_server.go

```diff
@@ -49,8 +49,10 @@ func runTPRServer(opts *ServiceCatalogServerOptions) error {
 	glog.Infoln("Installing TPR types to the cluster")
 	tprInstaller := tpr.NewInstaller(tprOpts.clIface.Extensions().ThirdPartyResources())
 	if err := tprInstaller.InstallTypes(); err != nil {
-		glog.V(4).Infof("Installing TPR types failed, continuing anyway (%s)", err)
+		glog.Errorf("Failed to installing TPR types (%s)", err)
+		return err
```

Comment:
per the interrupt catcher installed in https://github.com/kubernetes-incubator/service-catalog/blob/master/cmd/apiserver/app/server/server.go#L110